marklogic-client-api/src/main/java/com/marklogic/client/impl/XdmpExprImpl.java

```diff
@@ -1080,6 +1080,45 @@ public ServerExpression type(ServerExpression value) {
   }
 
   
+  @Override
+  public ServerExpression unquote(ServerExpression arg) {
+    if (arg == null) {
+      throw new IllegalArgumentException("arg parameter for unquote() cannot be null");
+    }
+    return new BaseTypeImpl.DocumentNodeSeqCallImpl("xdmp", "unquote", new Object[]{ arg });
+  }
```

Comment:
@rjrudin and @llinggit , the code currently does not compile since bringing back "unquote" requires us to add a new "DocumentNodeSeqCallImpl" in BaseTypeImpl. Any suggestions here?

Reply:
That seems fine to me, since BaseTypeImpl isn't generated. Though, do we just assume that DocumentNodeSeqCallImpl should have a constructor of `String, String, Object[]`?